index.html

```diff
@@ -3,15 +3,17 @@
 
 <head>
   <title>ZooRoyal</title>
+  <link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
Die Aufgabe war, das die CSS Datei in einem Unterordner mit dem Namen "css" abgelegt wird und dort aufgerufen wird:

"./css/style.css"